packages/app-page-builder/src/types.ts

```diff
@@ -119,7 +119,9 @@ export interface PbElementDataSettingsType {
     margin?: PbElementDataSettingsMarginType;
     padding?: PbElementDataSettingsPaddingType;
     height?: {
-        value?: number;
+        [key: string]: {
```

Comment:
Can you explain this change? Was this a bug? I believe so, but can you remind me?

Reply:
Ah good point ... it's not needed anymore, I was previously using 

```
const elementHeight = settings.height[displayMode].value
```

But now with dotProp's `get()` I can revert it.

Reply:
@endymion1818 I think this is not a bad thing have in types. But, it would be better to use the DisplayMode enum as a key, not `string`. If you just revert this back, you'll have `value?: number`, which is wrong, since you have that DisplayMode key.

Reply:
After discussion I've added this back as `[key in DisplayMode]` using the `DisplayMode` enum.